src/core/hyper-ref/index.js

```diff
@@ -305,10 +304,51 @@ export default class HyperRef extends PureComponent<Props, State> {
       });
     }
 
-    return React.createElement(
-      TouchableOpacity,
-      { ...props, ...pressHandlers, accessible: false },
-      children,
+    const style = this.getStyle();
+    const { accessibilityLabel, testID } = createTestProps(this.props.element);
+    const { onLongPress, onPress, onPressIn, onPressOut } = pressHandlers;
+
+    // If element is a <text> nested under another <text>, simply add press events
+    const isNestedUnderText =
+      this.props.element.parentNode?.namespaceURI === Namespaces.HYPERVIEW &&
+      this.props.element.parentNode?.localName === LOCAL_NAME.TEXT;
+
+    if (isNestedUnderText) {
+      const noop = () => {};
+      return (
+        <Text
+          accessibilityLabel={accessibilityLabel}
+          accessible={false}
+          onLongPress={onLongPress}
+          // when no press handler set, we still need an empty handler for pressIn or pressOut handlers to work
+          onPress={onPress || (onPressIn || onPressOut ? noop : undefined)}
```

Comment:
```suggestion
          onPress={onPress || ((onPressIn || onPressOut) ? noop : undefined)}
```

Reply:
hmm I guess prettier does not want these extra parenthesis and `?` is interpreted on the whole expression, not just the last bit, so change is not needed.